### PR TITLE
LPS-12345 | master

### PR DESCRIPTION
**Pull request by JingyuanZhao**
@Tim-Cao

### CODE REVIEW

**Comment by JingyuanZhao** [comment]
ci:test:sf

**Comment by JingyuanZhao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
Hi @JingyuanZhao
Remove this commit https://github.com/Tim-Cao/liferay-portal/pull/693/commits/eced2f138f502ba6678cd0038810ee074d2867b1 from your PR

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@component-name = "portal-smoke"
+definition {
+
+	property testray.main.component.name = "Smoke";
+
+	test TrainingSignIn {
+		User.firstLoginUI();
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:8]: Missing step to check sign in successfully.

File: portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@component-name = "portal-smoke"
+definition {
+
+	property testray.main.component.name = "Smoke";
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/training/Training.testcase:5]: Add @description and @priority label.
You could assume priority is 5

**Comment by Tim-Cao** [comment]
ci:help

**Comment by JingyuanZhao** [comment]
ci:test

**Pull request closed without merging**